src/static/current-state/components/QueueItem.tsx

```diff
@@ -259,11 +258,22 @@ export class QueueItem extends React.Component<QueueItemProps, QueueItemState> {
             </StatusItem>
           ))}
         </div>
-        {status.reason ? (
+        {status.reason && status.state !== 'success' ? (
           <div className="queue-item__status-line">
             <StatusItem title="Reason:">{status.reason}</StatusItem>
           </div>
         ) : null}
+        {status.request.triggererAaid ? (
+          <StatusItem title="Landed by:">
+            <Lozenge>
+              <User aaid={status.request.triggererAaid}>
+                {user => {
+                  return user.displayName;
+                }}
+              </User>
+            </Lozenge>
+          </StatusItem>
+        ) : null}
```

Comment:
I do like this, but the reason section will display this info when a request has been queued, perhaps you could change the status state check on the reason section to be `status.state !== 'queued'`?

Reply:
Hmm, I added a reason to a `success` status too, should I take this out since it'll show up twice? Or change the check so it's `status.state !== 'queued' && status.state !== 'success'`?

Reply:
Ahh didn't see that one, I would remove the `Landed by` reason from the `success` status transition, and change that check just to `status.state !== 'queued'`.